redaxo/src/core/fragments/core/header.php

```diff
@@ -7,6 +7,11 @@
                     <?php else: ?>
                         <a class="navbar-brand" href="<?= rex_url::backendController() ?>"><img class="rex-js-svg rex-redaxo-logo" src="<?= rex_url::coreAssets('redaxo-logo.svg') ?>" /></a>
                     <?php endif; ?>
+                    <?php if (rex::getUser()->isAdmin() && rex::isDebugMode()): ?>
+                        <a class="rex-marker-debugmode" href="<?= rex_url::backendPage('system/settings') ?>" title="<?= rex_i18n::msg('debug_mode_marker') ?>">
+                            <i class="rex-icon fa-heartbeat"></i>
```

Comment:
Wir verwenden keine fa-Icons direkt im Code, da diese ansonsten nicht austauschbar mit anderen Iconfonts sind.

Reply:
PR in #2577